src/PotentialSimulation/PotentialSimulationSetups/PotentialSimulationSetupRBCartesian3D.jl

```diff
@@ -60,6 +60,7 @@ function set_point_types_and_fixed_potentials!(point_types::Array{PointType, N},
             end
         end
     end
+    mark_surface_bits!(point_types)

```

Comment:
How long does it take to mark a grid (e.g. 300x300x300)?
We only need this for the final grid. So we might want to skip this step for the coarse grids.

Reply:
For the example `BEGe` with a grid dimensions `(206, 60, 310)`, it takes around 300ms.

```julia
using SolidStateDetectors
using BenchmarkTools
sim = Simulation(SSD_examples[:BEGe])
calculate_electric_potential!(sim, refinement_limits = [0.2,0.1,0.05,0.02,0.01])
point_types = SolidStateDetectors.get_2π_potential(PointTypes(sim.point_types.data .& 0x07, sim.point_types.grid), n_points_in_φ = 60).data
@btime SolidStateDetectors.mark_bulk_bits!($point_types);
```
```
306.982 ms (1916682 allocations: 249.05 MiB)
```
However, there should be a way to get this allocation-free.
The number of allocations is a bit more than half the number of grid points..

Reply:
After the latest commit (34ea3a8), the execution time is shorter:
```
133.343 ms (6 allocations: 40.26 MiB)
```